modules/gdscript/gdscript_editor.cpp

```diff
@@ -2848,16 +2848,6 @@ ::Error GDScriptLanguage::complete_code(const String &p_code, const String &p_pa
 				break;
 			}
 
-			GDScriptParser::CompletionContext c = completion_context;
```

Comment:
Or maybe instead of removal there should be fix like:
```
GDScriptParser::CompletionContext& c = completion_context;
```

Reply:
It's hard to say. The code was added as is in aa09b4f85d9 so it likely never did anything.